mozilla/activity-stream
# Preparing the release of Activity Stream 1.1.6

Hi all,

We are preparing the release of A-S 1.1.6. The add-on has been made available at [pre-release channel](https://moz-activity-streams-prerelease.s3.amazonaws.com/dist/latest.html), where all the release stakeholders can test and verify the add-on. That been said, we won't send a specific build as the release candidate anymore, instead, everyone should install the add-on from here during the release preview, it'll automatically update itself whenever engineer pushes bugfixes to repo.

This is our first attempt to smooth out the release process, you can see more details at [deploy.md](https://github.com/mozilla/activity-stream/blob/master/deploy.md). As usual, feedback and advises are always welcome.
# Release checklist
- QA
- Product
  - [x] @nchapman or @aaronrbenson 
- Engineering

Please use the `release/block` or `release/uplift` to label the bug report as well as other release related issues. Once you finish the review on your end, please check off the corresponding item in the above checklist.

Thanks
